src/mapbox_logo.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var MapboxLogo = L.Control.extend({
```

Comment:
Can this be called MapboxLogoControl to be more in line with the rest and clearly indicate that it's a control?